ivy/container/random.py

```diff
@@ -58,3 +58,57 @@ def random_uniform(
             map_sequences,
             out=out,
         )
+
+
+    # randint
+    @staticmethod
+    def static_randint(
+        low: Union[int, ivy.Container] = 0.0,
+        high: Union[int, ivy.Container] = 1.0,
+        shape: Optional[Union[int, Tuple[int, ...], ivy.Container]] = None,
+        device: Optional[Union[ivy.Device, ivy.NativeDevice, ivy.Container]] = None,
+        key_chains: Optional[Union[List[str], Dict[str, str]]] = None,
+        out: Optional[ivy.Container] = None,
+    ) -> ivy.Container:
+        """
+        ivy.Container static method variant of ivy.randint. This method simply wraps the
+        function, and so the docstring for ivy.randint also applies to this method
+        with minimal changes.
+
+        Examples
+        --------
+        With no device argument and no out argument specified and int arguemnt for shape:
+
+        >>> x = ivy.Container.randint(low=ivy.Container(a=1), high=ivy.Container(b=10), shape=2)
```

Comment:
Hi @donaldchan10, so the main advantage with a`ivy.Container` is that it enables us to process multiple values at once, meaning `ivy.Container.randint(low=ivy.Container(a=1), high=ivy.Container(b=10), shape=2)` should be replaced by something like `ivy.Container.randint(low=ivy.Container(a=1, b=3), high=ivy.Container(a=11, b=10), shape=2)` for the examples, because otherwise it would generate a `KeyError`. Please make that update and also update the result of the functions by running the examples locally.

Let me know if you have any questions. Thanks 🙂